hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -83,6 +84,15 @@ public void initialize(String inputName, Map<String, String> properties) {
       this.conf = new Configuration();
     }
 
+    // Forward all catalog properties that begin with `hive.` into configuration for use in setting
+    // up the
+    // MetastoreClient
```

Comment:
Looks like this was auto formatted and could be fixed.

Reply:
Will do